lambdas/data-migration2/src/storeErrors.ts

```diff
@@ -0,0 +1,34 @@
+import moment from 'moment';
```

Comment:
We're using moment but I don't see that we've added it as a dependency to the package.json for the Lambda?